icu4c/source/i18n/unicode/numberformatter.h

```diff
@@ -2041,6 +2044,45 @@ class U_I18N_API NumberFormatterSettings {
      */
     Derived scale(const Scale &scale) &&;
 
+    /**
+     * Specifies the usage of the unit ("person", "road", "person" ...etc.)
+     *
+     * NOTE:    `usage` will change the output unit depending on the `Locale`  
+     *          and the unit value.
+     *          For Example:
+     *              Locale: en_US
+     *              Usage : length-person
+     *              Unit  : Meter
+     *
+     *          If the unit value is 0.25, the output will be "10 inches."
+     *          If the unit value is 1.50, the output will be 
+     *                                           "4 feet and 11 inches"
+     *
+     * NOTE:    If the input usage is not exist (e.g. "dance") or is misspelled,
+     *          the usage will be **fall backed** to the "default" usage.
```

Comment:
Grammar.  Please fix to:

> NOTE: If the input usage does not exist (e.g. "dance") or is misspelled, the usage will fall back to the "default" usage.

Please also discuss the subtag stripping approach: "person-height" falls back to "person" before going to "default".